src/chains/types.ts

```diff
@@ -25,7 +49,23 @@ export type Precompile = {
   references: string[];
 };
 
+export type Opcode = {
+  number: number;
+  name: string;
+  description: string;
+  minGas: number;
+  gasComputation?: GasComputation;
+  inputs: Variable[];
+  outputs: Variable[];
+  examples: Example[];
+  playgroundLink?: string;
```

Comment:
Playground link is a great idea. Though we should probably filter this out when reusing the data for other chains, since the gas costs, etc. in the playground link don't necessarily apply until we have a dedicated playground for that chain (ref #5).

I'm thinking a function analogous to TS's [`Omit`](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys) or [type-fest's `Except`](https://github.com/sindresorhus/type-fest) would be useful here, i.e. pass it a list of opcodes and list of field names, and it deletes the specified field from each item.

Maybe a function similar to [`updateTodo`](https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype) called `updateOpcode` is useful too, to make it easy to redefine certain fields. Though I guess you can do that with spread syntax already, so maybe that doesn't add value.

Also for optimism where gas costs are the same, we can probably keep this link for most opcodes

Reply:
I added a new type `OpcodeWithoutPlaygroundLink` (see [7690723](https://github.com/mds1/evm-diff/pull/12/commits/7690723d40ebaf2e324e26db6f2529d896de8af3)). Feel free to modify the PR or suggest further changes :)